lib/modules/platform/pr-body.ts

```diff
@@ -53,10 +53,63 @@ export function getRenovateConfigHashPayload(body: string): string | undefined {
   return match?.groups?.payload;
 }
 
+function getRenovateBodyIndexes(input: string | undefined | null): {
```

Comment:
```suggestion
function getRenovateBodyIndexes(input: string): {
```
validate on caller function